### DISCUSSION TITLE
关于Logo

### DISCUSSION THREAD

**Discussion by Muro967** [ECUST-Leap/ecust-leap.github.io#118] [投票区]
大家觉得哪个logo更适合我们？
1号：
<img width="164" alt="image" src="https://user-images.githubusercontent.com/103167971/217163416-03846bbc-d879-4ce9-92b7-f2c91a802808.png">
2号：
<img width="164" alt="image" src="https://user-images.githubusercontent.com/103167971/217163779-74fa6f8b-a111-4f39-800d-5c6af6ec2d7c.png">
3号：
<img width="167" alt="image" src="https://user-images.githubusercontent.com/103167971/217163134-a166807d-a743-4287-83f5-8079c95b3476.png">
4号：
<img width="162" alt="image" src="https://user-images.githubusercontent.com/103167971/217163157-a905725c-6519-4a33-adb6-dee9ad52e0c4.png">
5号：
<img width="167" alt="image" src="https://user-images.githubusercontent.com/103167971/217163203-d8e1e5c8-00da-4804-99d1-9816dea6afd6.png">
6号：
<img width="176" alt="image" src="https://user-images.githubusercontent.com/103167971/217163313-ac7229a8-8721-479e-a4a4-66ffd1993234.png">
7号：
<img width="160" alt="image" src="https://user-images.githubusercontent.com/103167971/217163337-e7a03088-30c2-4d84-8391-54041746d7a0.png">
8号：
<img width="166" alt="image" src="https://user-images.githubusercontent.com/103167971/217163377-2ced5421-c583-4628-aa48-6995cb18cded.png">

**Comment by paulzrq** [comment]
有一种国足选拔的感觉qwq
